regen-network/regen-web
# feat: APP-366 account orders

## Description

https://regennetwork.atlassian.net/browse/APP-366




---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2566--regen-marketplace.netlify.app/
1. First, login and buy some credits with fiat and with crypto (older crypto orders below a certain block height are not available on staging because of state pruning on our redwood node, this won't be the case on mainnet since we use the archive node where we should be able to see old crypto orders)
2. Check out "my orders" and verify orders info there

~~**NB: credit card info is missing for now, I've realized that we need some server update for that, which I'm working on, but the rest can still be tested in the mean time.**~~ DONE

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
